<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addSolutionDlg</class>
 <widget class="QWidget" name="addSolutionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>154</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>409</width>
    <height>154</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>救援方案管理</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>解决方案路径</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="selectPathButton">
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="openSolutionPathButton">
      <property name="text">
       <string>显示解决方案</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeSolutionButton">
      <property name="text">
       <string>关闭解决方案</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
